Move 4 IE specific props under HTMLMediaElement (#16959)
mdn/content: files/en-us/web/api/htmlmediaelement/msplaytosource/index.md

---
title: HTMLMediaElement.msPlayToSource
slug: Web/API/HTMLMediaElement/msPlayToSource

> page-type: api-instance-property

Reviewer: ```suggestion
page-type: web-api-instance-property
```

This proprietary property is specific to Internet Explorer and Microsoft Edge.

## Value

> _PlayTo_ is a mean through which an app can connect local playback/display for

Reviewer: ```suggestion
_PlayTo_ enables an app can connect local playback/display for
```

("means to" is probably what was intended but it's perhaps not very clear)
